DataConnectors/CEF/cef_troubleshoot.py

```diff
@@ -383,6 +384,23 @@ def check_oms_agent_status():
     else:
         return True
 
+def check_omsagent_new_version(omsagent_plugin_securiy_config):

```

Comment:
if this is the name maybe we should really check the version of the agent and based our suggestion on that for future updates.